README.md

```diff
@@ -497,7 +497,7 @@ You can also view it on [Helm Hub](https://hub.helm.sh/charts/stable/chartmuseum
 
 To install:
 ```bash
-helm repo add stable https://kubernetes-charts.storage.googleapis.com
+helm repo add stable https://charts.helm.sh/stable
```

Comment:
@antoinedeschenes is correct. Can we switch this to
`helm repo add chartmuseum https://chartmuseum.github.io/charts` ?